scripts/testing/run_e2e_on_k3d.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# This script uses the following environment variables (if not provided script defines default values):
+#                       LOCAL_REGISTRY
+#                       K3D_REGISTRY
+#                       PR_NAME
+#                       MODULE_PREFIX
+# ./run_e2e_on_k3d.sh
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # needs to be set if we want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being maskedPORT=5001
+
+# registry access from local machine
+LOCAL_REGISTRY=${LOCAL_REGISTRY:-localhost:5001}
+
+#registry access from k3d cluster
+K3D_REGISTRY=${K3D_REGISTRY:-k3d-kyma-registry:5000}
+
+PR_NAME=${PR_NAME:-PR-176}
```

Comment:
hardcoded value

Reply:
I needed to pick some default value, I picked current PR number, but this is not a best choice. To reconsider what the default value should be.